src/js/mixins/QueryParamsMixin.jsx

```diff
@@ -79,6 +79,10 @@ var QueryParamsMixin = {
     }
 
     router.transitionTo(router.getCurrentPathname(), {}, queryParams);
+  },
+
+  decodeArray: function (array) {
```

Comment:
Hmm, the array _encoding_ is also generic-wise:
https://github.com/mesosphere/marathon-ui/blob/fix/broken-labels/src/js/mixins/QueryParamsMixin.jsx#L9
I would like to stick with the generic name. It is also more descriptive in a technical sense imho.
Maybe `decodeParamArrayValue` ?